prepare.js

```diff
@@ -0,0 +1,10 @@
+// download latest gitmojis.json
+const https = require('https');
+const fs = require('fs');
+
+const file = fs.createWriteStream('./src/vendors/gitmojis.json');
+const request = https.get('https://gitmoji.dev/api/gitmojis', function (
```

Comment:
```suggestion
const request = https.get('https://raw.githubusercontent.com/carloscuesta/gitmoji/v3.5.0/src/data/gitmojis.json', function (
```

I think a clear label can better ensure that the version is correct. If you need to make changes in the future, you can submit a new PR again to prevent any incompatibility issues with the latest version.  😁